src/libraries/System.Net.Http/src/System/Net/Http/Headers/CacheControlHeaderValue.cs

```diff
@@ -27,103 +26,128 @@ public class CacheControlHeaderValue : ICloneable
 
         private static readonly GenericHeaderParser s_nameValueListParser = GenericHeaderParser.MultipleValueNameValueParser;
 
-        private bool _noCache;
+        [Flags]
+        private enum Flags : int
```

Comment:
Nit: It doesn't currently look like it would help with anything, but this could be a {u}short.

Reply:
Left it as `int` for now to make `Interlocked` work